src/nlp/api.jl

```diff
@@ -161,6 +264,68 @@ function jac(nlp::AbstractNLPModel, x::AbstractVector)
   sparse(rows, cols, vals, nlp.meta.ncon, nlp.meta.nvar)
 end
 
+"""
+    vals = lin_jac_coord!(nlp, x, vals)
+
+Evaluate ``J(x)``, the linear constraint's Jacobian at `x` in sparse coordinate format,
+rewriting `vals`.
```

Comment:
```suggestion
overwriting `vals`.
```

Reply:
I also changed the others in the same file.